tests/jsmarionette/client/marionette-client/lib/marionette/client.js

```diff
@@ -473,11 +454,10 @@
       var args = Array.prototype.slice.call(arguments),
           callback = args.shift();
 
-      if (!callback) {
```

Comment:
Why is `callback` being changed to `cb` here? The variable is still `callback` on L455?